react-native-gutenberg-bridge/ios/GutenbergBridgeDelegate.swift

```diff
@@ -64,6 +64,25 @@ extension RCTLogLevel {
     }
 }
 
+public enum GutenbergUserEvent {
+    
+    case editorSessionTemplateApply(_ template: String)
+    case editorSessionTemplatePreview(_ template: String)
```

Comment:
🤩 Not sure if this is new to me or I have forgotten my Swift already, but it's cool that you can label associated values.